src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -625,6 +639,31 @@ namespace AppInstaller::Manifest
         return resultErrors;
     }
 
+    ValidationErrors ManifestYamlPopulator::ProcessExpectedReturnCodesNode(const YAML::Node& returnCodesNode)

```

Comment:
>ProcessExpectedReturnCodesNode [](http://example.com/codeflow?start=44&length=30)

As of convention and completeness, please add the new manifest entries in MultiFileManifestV1_1 and ManifestV1_1-Singleton.yaml and update the VerifyManifestContent in unit tests.